index.html

```diff
@@ -26,6 +26,8 @@
         
         
     </header>
+
```

Comment:
[OPTIONAL]You did a brilliant job but let's make our web page more responsive. Kindly check the bug, I shared below on a tablet screen view. 📱

<img width="1148" alt="Screenshot 2023-02-02 at 11 33 38 PM" src="https://user-images.githubusercontent.com/25887253/216444280-dffd2960-9c82-4f2b-9a73-3e8b6362c2dd.png">

Reply:
Okay thank you, I fix that in the next commit ... thanks a lot